tests/installation/start_install.pm

```diff
@@ -48,6 +48,7 @@ sub run {
                 while (!check_screen('all-conflicts-resolved-packages', 4)) {
                     assert_and_click 'package-conflict-choice';
                     send_key $cmd{ok};
+                    wait_still_screen 10;
```

Comment:
Yeah, idea was that I provide a fix in couple of hours, but you're right, let's do better solution now.